java/play-a-kml-tour/README.md

```diff
@@ -0,0 +1,76 @@
+# Play a KML tour
+
+Play tours in KML files.
+
+![Play a KML tour App](play-a-kml-tour.png)
+
+## Use case
+
+KML, the file format used by Google Earth, supports creating tours,
+which can: control the viewpoint of the scene, hide and show content,
+and play audio. Tours allow you to easily share tours of geographic
+locations, which can be augmented with rich multimedia. Runtime allows
+you to consume these tours using a simple API.
+
+## How to use the sample
+
+The sample will load a KMZ file. When a tour is found, the buttons will
+be enabled. Use _Play_ and _Pause_ to control the tour. When you're
+ready to show the tour, use _Reset_ to return the tour to the unplayed
+state.
+
+## How it works
+
+1. Load the KML file and add it to a layer. 
+2. Create the KML tour controller. Wire up the buttons to the `Play`,
+   `Pause`, and `Reset` methods. 
+1. Explore the tree of KML content and find
+   the first KML tour. Once a tour is found, provide it to the KML tour
+   controller. 
+4. Enable the buttons to allow the user to play, pause, and reset the
+   tour.
+   
+## Relevant API
+
+* KmlTourController
+* KmlTourController.Tour
+* KmlTourController.Play()
```

Comment:
Should we be including the methods used here and not just the classes? I haven't seen this in other samples.

Reply:
Its in the design, so I think we'll just leave it. The alternative is to have KmlTourController and KmlTour only, which maybe isn't so helpful.